### ISSUE TITLE
Filepath changes caused asset deduplication to fail on non-Windows platforms

### ISSUE THREAD

**Issue by alexanderbock** [OpenSpace/OpenSpace#1606]
When loading a profile:
```
2021-05-24	11:20:52.870	Asset	Debug	Initializing asset /Users/dev/openspace/OpenSpaceMaster/data/assets/scene/solarsystem/sun/./transforms.asset
...
2021-05-24	11:20:52.875	Asset	Debug	Initializing asset /Users/dev/openspace/OpenSpaceMaster/data/assets/scene/solarsystem/sun/./layers/colorlayers/./../.././transforms.asset
```

**Comment by GPayne** [comment]
I do not see this problem on linux running the master branch.

**Comment by alexanderbock** [comment]
Great, then it is fixed as of: 6cf6e450fbbc461ee2bfb7cfb958286ca8c6b0f3

The problem was that the `absPath` function no longer lexically normalized the path, which has now been fixed
